masonry/src/text/edit.rs

```diff
@@ -75,7 +75,7 @@ impl<T: EditableText> TextEditor<T> {
 
     /// Rebuild the text.
     ///
-    /// See also [TextLayout::rebuild](crate::text2::TextLayout::rebuild) for more comprehensive docs.
+    /// See also [`TextLayout::rebuild()`](crate::text::TextLayout::rebuild) for more comprehensive docs.
```

Comment:
I'm not really sure I like the function call brackets here. E.g. https://doc.rust-lang.org/stable/std/vec/struct.Vec.html#indexing doesn't use that form, unless it cares about the value of the argument.

Reply:
I tend to do them partly out of a habit from places where you might refer to either a member or a function, and from Doxygen which auto-links things that use `()` ... but I have got them into lots of Rust code now!

Reply:
I think it makes sense to stay consistent, and I don't know of crates that use `()` for doc-links.